spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1693,7 +1693,10 @@ properties (with the prefix stripped) when the local `EntityManagerFactory` is c
 
 [[howto-configure-hibernate-naming-strategy]]
 === Configure Hibernate Naming Strategy
-Hibernate defines `Physical` and `Implicit` naming strategies. Spring Boot configures
+Since version 5, Hibernate defines `Physical` and `Implicit` naming strategies which you
```

Comment:
Even though working for quite a while with Hibernate 5, I wasn't aware of the implicit and explicit naming strategies. Knowing the old property `spring.jpa.hibernate.naming.strategy`, I was confused as to what implicit and explicit naming strategies meant until I realized that they came with Hibernate 5 and replaced the old property. Thus, the addition of "since version 5" would have helped me get oriented sooner.